consensus/src/consensusdb/mod.rs

```diff
@@ -76,16 +79,17 @@ impl ConsensusDB {
     pub fn save_highest_timeout_certificate(
         &self,
         highest_timeout_certificate: Vec<u8>,
-    ) -> Result<()> {
+    ) -> Result<(), DbError> {
         let mut batch = SchemaBatch::new();
         batch.put::<SingleEntrySchema>(
             &SingleEntryKey::HighestTimeoutCertificate,
             &highest_timeout_certificate,
         )?;
-        self.commit(batch)
+        self.commit(batch)?;
+        Ok(())
```

Comment:
awww... why not `self.commit(batch).map_err(|e| e.into())` that is if the type is different